terraform/environment/terraform.tfvars.json

```diff
@@ -53,8 +53,8 @@
       "account_id": "805626386523",
       "is_production": true,
       "aurora_serverless_auto_pause": true,
-      "database_deletion_protection": true,
-      "skip_final_snapshot": false,
+      "database_deletion_protection": false,
```

Comment:
It'd be useful to know the reasons why these two have flipped.

Reply:
this was preventing deletion of the final db, and we took a manual snapshot as no-one had access. missed that on previous code, so this is to tidy that up (also an explanation - sorry 😁  )